turftopic/models/cluster.py

```diff
@@ -95,6 +98,24 @@ def calculate_topic_vectors(
     return centroids
 
 
+def build_tsne(*args, **kwargs):
+    try:
+        from openTSNE import TSNE
+
+        model = TSNE(*args, **kwargs)
+        model.fit_transform = model.fit
+        return model
+    except ModuleNotFoundError:
+        from sklearn.manifold import TSNE
+
+        warnings.warn(
```

Comment:
So by default, I would get a warning? sounds unpleasant

Reply:
I mean, it totally is unpleasant. I think the thing to consider here is:
*What is our users' experience more hindered by? Having a TSNE implementation that's orders of magnitudes slower, or having to see a warning when they create clustering topic models?*

Considering, how people use libraries (usually they play around with stuff before giving a proper go at the documentation, and really only read it when they get stuck). There is absolutely no better way to communicate this.
They can also easily make this message disappear by specifying, which clustering model they really want.

Reply:
I would specify the last part in the message